### PR TITLE
docs: adding explainer page about Dev card

### LINKED ISSUE #156: Feature: Add Dev Cards to Getting Started
### Type of feature

🍕 Feature

### Current behavior

Currently, we don't have any documentation on Dev cards. We need to add a section.

### Suggested solution

Let's highlight our Dev Card feature: https://insights.opensauced.pizza/user/bekahhw/card

For each section in the Getting Started category, we need to keep in mind the following:
1. Features are for maintainers and contributors at all stages of their journey. Descriptions should be inclusive of that audience.
2. The first part of the feature documentation should explain what the feature is and why it's useful.
3. There should be a screenshot or gif that demonstrates what the feature is or what it does.
4. If necessary, explain why it's important to contributors and maintainers.

Keeping that in mind, let's add Dev Card to the Getting Started panel.

## How to Implement this issue
1. In the repository, go to `docs/community`
2. Add a file called `dev-card.md`
3. In `sidebar.js`, add "community/dev-card", to the `items` array.
4.  In `dev-card.md` replicate the frontmatter in any of the other community files (frontmatter is the part container between the ---) but replace with devcard and update the description.
5. Within that file, create the copy to using the guidelines above.

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by CBID2**
## Description
This PR adds an explainer page about the dev card feature. 

## What type of PR is this? (check all applicable)

- [X] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Closes #156 
## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by CBID2** [comment]
Hey @BekahHW. Here are the screenshots I came up with for the devcard.
## Front
front-of-dev-card

## Back
back-of-dev-card

What do you think?

**Comment by BekahHW** [comment]
@CBID2 let's use screen shots of only the card. Similar to this:

Bekah Dev Card

@isabensusan is there a png or SVG of the front and back of the dev card that we can use here instead of a screenshot?

File: docs/community/dev-card.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+id: dev-card
+title: 'Showcase your open source stats with your dev card'
+sidebar_label: 'Dev Card'
+keywords:
+- dev-card 
+---
+
+The Dev card feature is what we like to call your "open source business card". At the front, you see your username, the amount of pull requests you create, repositories you contribute to, a `View Profile` button that takes you to your dashboard, and a green icon, which describes your activity rate.
```
Comment (BekahHW) [inline docs/community/dev-card.md:9]: Move the period inside the quotes.

Comment (BekahHW) [reply docs/community/dev-card.md:9]: change "amount" to "number"

File: docs/community/dev-card.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+When you click on the back of your card, you see the same information that's shown in the front as well as the headline from your OpenSauced profile, amount of time you've been using our product, and your [PR velocity rate](../../docs/community/pr-velocity.md).
```
Comment (BekahHW) [inline docs/community/dev-card.md:11]: Let's change "you see the same information" to "you see additional information, including ..."

I believe that the date isn't when you started using the product. I think it's your first commit. @bdougie can you verify that?

File: docs/community/dev-card.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+[screenshot of back of the dev card]().
+
+Whether you are a person who's just beginning their open source journey or a maintainer looking for more contributors and/or sponsors, the dev card is a great way to show your audience how active you are in the community.
```
Comment (BekahHW) [inline docs/community/dev-card.md:14]: Maybe add something like this:

The Dev Card is a shareable snapshot of your contributions, skills, and engagement in the open source community.

**Comment by BekahHW** [comment]
Thanks for all your work on this, @CBID2. Sorry it's taken so long for me to get to it,

**Comment by BekahHW** [comment]
@CBID2 here's some SVGs you can use: https://drive.google.com/drive/folders/1NAyG4S9luQv9VovHpel9cReRUEWGFfFz?usp=sharing

**Comment by CBID2** [comment]
> @CBID2 here's some SVGs you can use: https://drive.google.com/drive/folders/1NAyG4S9luQv9VovHpel9cReRUEWGFfFz?usp=sharing
> 
> 

Added them :)

File: docs/community/dev-card.md (hunk 2 above)
Comment (BekahHW) [reply docs/community/dev-card.md]: Ok, that date is the first contribution date on github

File: static/img/Group 113.svg
Comment (BekahHW) [inline static/img/Group 113.svg:1]: Let's rename the files so it's clearer for contributors who come in after what the images are. For example, "dev-card-front.svg"

File: docs/community/dev-card.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+[screenshot of front of a dev card](../../static/img/Group%20113.svg)
+
+When you click on the back of your card, you see additional information, including your OpenSauced profile's headline, amount of time you've been using our product, and your [PR velocity rate](../../docs/community/pr-velocity.md).
+[screenshot of back of a dev card](../../static/img/devcard%20vbase%20back.svg).
+
+Whether you are a person who's just beginning their open source journey or a maintainer looking for more contributors and/or sponsors, the Dev Card is a shareable snapshot of your contributions, skills, and engagement in the open source community.
+
+Eager to get started? Check out the [Discovering Open-Source with OpenSauced](https://github.com/open-sauced/intro/blob/main/05-how-to-contribute-to-open-source.md#discovering-open-source-with-opensauced) section in our free beginner course on open source to create your account.
```
Comment (BekahHW) [inline docs/community/dev-card.md:17]: I think the call to action (CTA) here needs to be, "Eager to get your Dev Card? Create it today and share with your network!"

File: docs/community/dev-card.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+id: dev-card
+title: 'Showcase your open source stats with your dev card'
+sidebar_label: 'Dev Card'
+keywords:
+- dev-card 
+---
+
+The Dev card feature is what we like to call your "open source business card." At the front, you see your username, the number of pull requests you have created, repositories you contribute to, a `View Profile` button that takes you to your dashboard, and a green icon, which describes your activity rate.
+[screenshot of front of a dev card](../../static/img/Group%20113.svg)
```
Comment (BekahHW) [inline docs/community/dev-card.md:10]: Instead of linking the images, display them on the page.

File: docs/community/dev-card.md (hunk 4 above)
Comment (BekahHW) [reply docs/community/dev-card.md:17]: I think, in general, for each of our features that gets documented, the CTA should be for the user to utilize the feature. If we have other materials on how to use it, then linking to that information is good.

**Review by BekahHW — CHANGES_REQUESTED** [review]

File: docs/community/dev-card.md (hunk 5 above)
Comment (CBID2) [reply docs/community/dev-card.md:10]: How do I go about doing that @BekahHW?
Like this?:
```
<img src="">
```

Comment (BekahHW) [reply docs/community/dev-card.md:10]: You can do that or add an ! in front of the markdown format alt description

Comment (CBID2) [reply docs/community/dev-card.md:10]: Ahh ok

Comment (CBID2) [reply docs/community/dev-card.md]: Done @BekahHW

File: docs/community/dev-card.md (hunk 4 above)
Comment (CBID2) [reply docs/community/dev-card.md]: > I think, in general, for each of our features that gets documented, the CTA should be for the user to utilize the feature. If we have other materials on how to use it, then linking to that information is good.

I like that rule! :) It'll further encourage users to learn how to use the product.

File: docs/community/dev-card.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+sidebar_label: 'Dev Card'
+keywords:
+- dev-card 
+---
+
+The Dev card feature is what we like to call your "open source business card." At the front, you see your username, the number of pull requests you have created, repositories you contribute to, a `View Profile` button that takes you to your dashboard, and a green icon, which describes your activity rate.
+![screenshot of front of a dev card](../../static/img/Group%20113.svg).
+
+When you click on the back of your card, you see additional information, including your OpenSauced profile's headline, amount of time you've been using our product, and your [PR velocity rate](../../docs/community/pr-velocity.md).
+! [screenshot of back of a dev card](../../static/img/devcard%20vbase%20back.svg).
```
Comment (BekahHW) [inline docs/community/dev-card.md:13]: There's a space between the ! and [ that needs to be removed

Comment (CBID2) [reply docs/community/dev-card.md]: Done @BekahHW

**Review by BekahHW — CHANGES_REQUESTED** [review]
A couple of things.

1. When renaming files, make sure to follow the naming conventions. For example, it would be `devcard-back.svg`, notice the hyphen in there. 
2. The second svg didn't get renamed.
3. Running this locally, the cards take up too much space. We can make them smaller, but I'm also thinking that creating a new image that has them both together in one svg would be better, so they'd be side by side. That would also reduce chances of weirdness with different screen sizes and the text before and after the images. What do you think?

**Comment by CBID2** [comment]
> A couple of things.
> 
> 
> 
> 1. When renaming files, make sure to follow the naming conventions. For example, it would be `devcard-back.svg`, notice the hyphen in there. 
> 
> 2. The second svg didn't get renamed.
> 
> 3. Running this locally, the cards take up too much space. We can make them smaller, but I'm also thinking that creating a new image that has them both together in one svg would be better, so they'd be side by side. That would also reduce chances of weirdness with different screen sizes and the text before and after the images. What do you think?

That would be helpful @BekahHW! :) The images are big. What size do you recommend?

**Comment by BekahHW** [comment]
@CBID2 try the new image I put in the drive folder to see if that works https://drive.google.com/drive/folders/1NAyG4S9luQv9VovHpel9cReRUEWGFfFz

**Comment by CBID2** [comment]
> @CBID2 try the new image I put in the drive folder to see if that works https://drive.google.com/drive/folders/1NAyG4S9luQv9VovHpel9cReRUEWGFfFz

It looks so much better @BekahHW! :)

**Comment by BekahHW** [comment]
@CBID2 I added a new image to the drive file that sizes the cards more evenly. Can you use that? 

Last thing: There's an extra period in there: 
image

**Comment by CBID2** [comment]
> @CBID2 I added a new image to the drive file that sizes the cards more evenly. Can you use that? 
> 
> 
> 
> Last thing: There's an extra period in there: 
> 
> image
> 
> 

Sure @l

**Comment by CBID2** [comment]
Made the changes @BekahHW! :)

**Review by BekahHW — APPROVED** [review]
LGTM!

**Comment by CBID2** [comment]
> LGTM!

Woohoo! Thanks @BekahHW! :)

**Pull request merged**